docker-compose.yml

```diff
@@ -16,9 +17,13 @@ services:
   server:
     build: .
     command: bash -l -c "npm install && npm start"
+    environment:
+      - FASTIFY_PORT=5100
+      - FASTIFY_ADDRESS=0.0.0.0
```

Comment:
We should be able to set the development `DATABASE_URL` here as `postgresql://postgres@db/app`, which uses the default `postgres` user which can automatically be authenticated because of the `POSTGRES_HOST_AUTH_METHOD=trust` setting above. So no "credentials" are leaked (even though it would have just been development/test credentials anyway).

Reply:
we do not need these env var, I'll remove them. 
I was using them because I was having trouble connecting to 5000 not realizing that it was the vite port and that fastify was running on 5100